pkg/scheduler/plugins/loadaware/load_aware.go

```diff
@@ -102,6 +103,9 @@ func (p *Plugin) Filter(ctx context.Context, state *framework.CycleState, pod *c
 
 	nodeMetric, err := p.nodeMetricLister.Get(node.Name)
 	if err != nil {
+		if errors.IsNotFound(err) {
```

Comment:
please add some comments like:

For nodes that lack load information, fall back to the situation where there is no load-aware scheduling, and deal with the scenario where the user installs the koordlet component according to the grayscale of the nodes in the cluster.

Reply:
done